sktime/transformations/series/detrend/_detrend.py

```diff
@@ -38,6 +38,9 @@ class Detrender(BaseTransformer):
         The forecasting model to remove the trend with
             (e.g. PolynomialTrendForecaster).
         If forecaster is None, PolynomialTrendForecaster(degree=1) is used.
+    model : {"additive", "multiplicative"}, default="additive"
```

Comment:
could you add more detail on this?

I.e., explain that the forecaster is fitted on the original data etc?

Reply:
Of course! I've added more detail. Let me know what you think!